docs/topics/i18n/translation.txt

```diff
@@ -682,6 +682,17 @@ be retrieved (and stored) beforehand::
     This is a URL: {{ the_url }}
     {% endblocktrans %}
 
+If you'd like to retrieve a translated string without displaying it, you can
+use the following syntax::
+
+    {% blocktrans asvar the_title %}The title is {{ title }}.{% endblocktrans %}
+
+    <title>{{ the_title }}</title>
+    <meta name="description" content="{{ the_title }}">
+
+In practice you'll use this to get strings that are used in multiple places
```

Comment:
Just to note I also therefore changed the same text used under `trans` in the same file to match this, hope that's okay.